shared/paths.yaml

```diff
@@ -2,10 +2,10 @@
 # path names must end with "/" and not include the name of the executable
 # 'workflow_tools' is set on runtime (see common_functions.py:load_paths())
 # paths referring to a local folder
-#R_path: "/package/R-3.3.1/bin/"
-#R_libs_path: "/data/manke/repository/scripts/snakemake_workflows/R/x86_64-unknown-linux-gnu-library/3.3/"
+#R_path: "/package/R-3.5.0/bin/"
+#R_libs_path: "/data/processing4/R_LIB/3.5/"
 #histoneHMM_path: "/data/manke/repository/scripts/snakemake_workflows/R/x86_64-redhat-linux-gnu-library/3.3/histoneHMM/bin/"
-# paths referring to /package/
+#paths referring to /package/
 #bedtools_path: "/package/bedtools2-2.25.0/bin/"
```

Comment:
None of those lines are even used any more...

Reply:
LOL I don't know why I removed spaces from a commented line :D

Reply:
A commented line in a file that can be deleted now...